fr_FR-7.6.bak/Introduction/Index.rst

```diff
@@ -69,11 +69,9 @@ et soumettre une demande de récupération via GitHub .
 Sinon, vous pouvez simplement signaler le problème en utilisant le bug tracker sur
 : https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-Editors/issues.
 
-Pour participer à la traduction dans d'autres langues voir
-`http://wiki.typo3.org/TYPO3_Tutorial_for_Editors
-<http://wiki.typo3.org/TYPO3_Tutorial_for_Editors>`_
+Pour participer à la traduction dans d'autres langues voir https://wiki.typo3.org/TYPO3_Tutorial_for_Editors.
```

Comment:
The wiki doesn't exist anymore, link to https://docs.typo3.org/m/typo3/tutorial-editors/7.6/en-us/ instead

Reply:
See above.